### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by coding-sue-true**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Javascript Calculator</title>
+    <!-- <script type="text/javascript" src="js/main.js"> </script> -->
```
Comment (JSChilds) [inline index.html]: You left this commented-out, so the calculator doesn't run!

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+    if (final_result_metric <= 18.5) {
+      alert("A bit too skinny...?");
+    } else if (final_result_metric <= 24.9) {
+      alert("Looking like a super model!");
+    } else if (final_result_metric <= 29.9) {
+      alert("I bet there's a gym very close to you!");
+    } else {
+      alert("I would be very careful and try contact a doctor...!");
+    };
```
Comment (JSChilds) [inline js/main.js]: You don't need semicolons after a }

**Review by JSChilds — CHANGES_REQUESTED** [review]
Code is really well layed-out and readable and it looks like it worked great, but the commented-out script tag means the calculator can't run. Keep an eye on these little details - it doesn't matter how great your code is if it doesn't run!

**Comment by coding-sue-true** [comment]
Everything was now revised.

**Review by JSChilds — APPROVED** [review]

**Pull request merged**